.github/workflows/build-deploy-stage.yml

```diff
@@ -0,0 +1,100 @@
+name: Deploy to Amazon ECS stage environment
+
+on:
+  push:
+    branches: [ "dev-branch-1", "main", "add-github-actions-cicd-pipeline" ]
```

Comment:
Not sure we have a branch called `"add-github-actions-cicd-pipeline"`

Reply:
same here - removed as well